a11y-meta-display-guide/2.0/draft/guidelines/index.html

```diff
@@ -593,15 +596,16 @@ <h3 data-localization-id="visual-adjustments-title">Visual
 				clipped by its container).</p>
 
 			<section id="va-examples">
-				<h4>Examples</h4>
+				<h5>Examples</h5>
 
 				<p>The examples are provided as lists of possible
 					descriptive and compact explanations for
 					flexibility of adoption.</p>
 
 				<aside class="example" title="Descriptive explanations">
+				<h5 data-localization-id="ways-of-reading-title">Ways of reading</h5>
```

Comment:
Why are we repeating the same title <h3> was Ways of reading, now we have a <h5> with the same heading "Ways of reading"?